app/presenters/admin/user_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class Admin::UserPresenter < SimpleDelegator
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers
+  include ActionView::Helpers::TranslationHelper
+  include ActionView::Context
+
+  def initialize(user)
+    @user = user
+    super(@user)
+  end
+
+  def turn_admin_button(is_admin)
+    translation_scope = 'admin.users.index'
+
+    if is_admin
+      btn_text = t('users.admin_off')
+      btn_class = 'btn-warning'
+      confirm_message = t('are you sure you want to remove administration rights from this user', scope: translation_scope)
+    else
+      btn_text = t('users.admin_on')
+      btn_class = 'btn-primary'
+      confirm_message = t('are you sure you want to give administration rights to this user', scope: translation_scope)
+    end
```

Comment:
hahaha, there would be no problem in having an
```
<% if current_team.admin?(user) %>
  ...
<% else %>
  ...
<% end %>
```
on the partial? :) I've kept it on Presenter to avoid the conditional on the view...